File: ReactMeals_WebApi/DTO/ManagementResponseDTO.cs
using System.Text.Json.Serialization;

namespace ReactMeals_WebApi.DTO
{
    public record ManagementResponseDTO(
    [property: JsonPropertyName("access_token")] string AccessToken,
    [property: JsonPropertyName("expires_in")] int ExpiresIn,
    [property: JsonPropertyName("scope")] string Scope,
    [property: JsonPropertyName("token_type")] string TokenType);
}

File: src/main/java/gr/jimmys/jimmysfoodzilla/dto/ManagementResponseDTO.java
package gr.jimmys.jimmysfoodzilla.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class ManagementResponseDTO {
    @JsonProperty("access_token")
    private String accessToken;

    @JsonProperty("expires_in")
    private int expiresIn;

    private String scope;

    @JsonProperty("token_type")
    private String tokenType;
}
